sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -130,6 +130,115 @@ To update the value of any of the the available settings, we will call the Updat
 Setting updatedSetting
    = settingsClient.UpdateSetting(settingsList.Value[0].Name, options).Value;
 ```
+## Creating a BackupRestoreClient
+
+To create a new `BackupRestoreClient` to perform these operations, you need the endpoint to an Azure Key Vault HSM and credentials.
```

Comment:
Does it make sense to name this the `BackupRestoreClient`. I see python and .NET refer to it as the `BackupClient`

Reply:
that was the name generated by the corde gen, and it makes sense since it performs not only backup but also restore , backup client would imply restore is done by someone else